Tutorial.playground/Contents.swift

```diff
@@ -0,0 +1,658 @@
+// Copyright (c) 2012-2014 Gabriel Gonzalez
```

Comment:
Do we want to add to this copyright, or does @Gabriel439 still have sole primary control of this tutorial?  I'm not sure what the legal status is here.